### ISSUE TITLE
Use client authenticated SSL talking to SCS

### ISSUE THREAD

**Issue by ahelsing** [GENI-NSF/geni-tools#757]
Xi says:
```
I have also added SSL wrapper for the SCS. You will use the following URL for secure connection.
https://oingo.dragon.maxgigapop.net:8443/geni/xmlrpc
The service now trusts client certs signed by GENI CH, GpENI BBN and Emulab.
The non-secure / plain-http port (8081) is still open. Once all clients can do SSL, we can close the non-secure port.
```

 _Imported from trac ticket [#757](http://trac.gpolab.bbn.com/gcf/ticket/757),  created by ahelsing on 11-20-2014 at 08:48, last modified: 11-20-2014 at 15:44_

**Comment by ahelsing** [comment]
Xi is actually trusting the 4 roots at http://groups.geni.net/geni/wiki/GeniTrustAnchors

_Trac comment by ahelsing on 11-20-2014 at 13:31_

**Comment by ahelsing** [comment]
SCS now accepts -h or -?

SCS now takes --key and --cert

Changed the default SCS URL to the new 8443 URL on oingo. Updated comments in SCS.py on other SCS URLs.

Modified calling of SCS to supply cert/key. stitchhandler also handles common SCS errors having to do with bad URLs.

Possible future enhancements:
 - use dossl to make SCS calls so we get more standard error handling
 - make scs.py do the error handling, so when used as a main we get the benefit
 - allow scs.py to use the -c / -f options to get the cert/key from your omni_config file.

_Trac comment by ahelsing on 11-20-2014 at 15:35_

**Comment by ahelsing** [comment]
This fixes #346

For testing, you can try:
 - scs.py with no args (should fail)
 - scs.py with just cert/key
 - scs.py with --scsurl of http/oingo/8081 (should still work)
 - scs.py with illyrica cert/key (should fail)
 - stitcher normal run with --noReservation
 - stitcher normal run using your illyrica cert/key --noReservation (should fail)
 - stitcher normal run with --scsURL to http / oingo / 8081 --noReservation (should work)

_Trac comment by ahelsing on 11-20-2014 at 15:44_
